architecture/architecture-component-builds.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
```

Comment:
It looks like we can update existing modules and include them here:

include::modules/builds-about.adoc[leveloffset=+1]
include::modules/builds-strategy-docker-build.adoc[leveloffset=+1]
include::modules/builds-strategy-s2i-build.adoc[leveloffset=+1]
include::modules/builds-strategy-custom-build.adoc[leveloffset=+1]
include::modules/builds-strategy-pipeline-build.adoc[leveloffset=+1]

We also have a module for BuildConfigs (that may need to be updated as well): 
include::modules/builds-buildconfig.adoc[leveloffset=+1]